mobile/src/components/app/app.tsx

```diff
@@ -1,15 +1,17 @@
 import React, { FC } from 'react';
 import { NavigationContainer } from '@react-navigation/native';
 import { GestureHandlerRootView } from 'react-native-gesture-handler';
+import RNBootSplash from 'react-native-bootsplash';
 import 'fast-text-encoding';
-
 import { Root as RootNavigation } from '~/navigation/root/root.navigation';
 import { styles } from './styles';
 
 const App: FC = () => {
   return (
     <GestureHandlerRootView style={styles.rootView}>
-      <NavigationContainer>
+      <NavigationContainer
+        onReady={(): Promise<void> => RNBootSplash.hide({ fade: true })}
```

Comment:
It's better not to use anonymous functions inside jsx, move it to a separate handler